lib/fictium/exporters/open_api/v3_exporter/content_formatter.rb

```diff
@@ -2,18 +2,26 @@ module Fictium
   module OpenApi
     class V3Exporter
       class ContentFormatter
+        APP_JSON = 'application/json'.freeze
+
         def format(http_object, default = nil)
           type = (http_object.presence && http_object[:content_type].presence) || default
           return if type.blank?
 
           {}.tap do |content|
             media_type = {
-              example: http_object[:body]
+              example: body_formatter(http_object[:content_type], http_object[:body])
             }
             media_type[:schema] = http_object[:schema] if http_object[:schema].present?
             content[type.to_sym] = media_type
           end
         end
+
+        protected
+
+        def body_formatter(content_type, body)
+          content_type == APP_JSON && !body.empty? ? JSON.parse(body) : body
```

Comment:
When content_type be APP_JSON and with an empty body, it's ok
```rb
irb(main):001:0> true && []
=> []
```
But when content_type not to be APP_JSON this method returns false
```rb
irb(main):002:0> false && []
=> false
```
And then, in line above, you are setting
```rb
media_type[:example] = false
```
Instead of
```rb
media_type[:example] = []
```
It's ok?

Reply:
@lucasVoboril  in case, that this method returns false, looks like this
```ruby
pry(main)> false && !body.empty? ? JSON.parse(body) : body
=> ""
```
never returns a boolean

Reply:
You are on right!